c-dependencies/js-compute-runtime/xqd.h

```diff
@@ -93,6 +93,49 @@ typedef struct DynamicBackendConfig {
 
 #define INVALID_HANDLE (UINT32_MAX - 1)
 
+typedef enum class FastlyStatus {
```

Comment:
I would put this definition in a different file, like our `host_call.h` header: `enum class` isn't valid C, and this declaration is inside the `extern "C" {` block started at the top of the header.

Defining it in a different header would have a nice side effect of allowing us to use c++ features a bit more freely :)

Reply:
Good point, I've moved it to host_call now 👍